fairscale/utils/golden_testing_data.py

```diff
@@ -6,6 +6,7 @@
 
 """ Golden data used in unit tests. """
 
+# FIXME: why is this in the main lib, rather than the unittests directory?
```

Comment:
we don't import anything from `tests` and we didn't create a `fairscale.testing` directory. I think we should do that. We only have `fairscale.utils.testing.py`. perhaps we should have a fairscale.testing dir and put all testing related code, like this one, there. what do you think?